File: src/main/java/pl/yalgrin/playnite/simplesync/exception/ManualSynchronizationRequiredException.java
package pl.yalgrin.playnite.simplesync.exception;

import java.io.Serial;

public class ManualSynchronizationRequiredException extends RuntimeException {

    @Serial
    private static final long serialVersionUID = -6189953396806395343L;

    public ManualSynchronizationRequiredException() {
    }

    public ManualSynchronizationRequiredException(String message) {
        super(message);
    }

    public ManualSynchronizationRequiredException(String message, Throwable cause) {
        super(message, cause);
    }

    public ManualSynchronizationRequiredException(Throwable cause) {
        super(cause);
    }

    public ManualSynchronizationRequiredException(String message, Throwable cause, boolean enableSuppression,
                                                  boolean writableStackTrace) {
        super(message, cause, enableSuppression, writableStackTrace);
    }
}

File: src/main/kotlin/pl/yalgrin/playnite/simplesync/exception/ManualSynchronizationRequiredException.kt
package pl.yalgrin.playnite.simplesync.exception

import java.io.Serial

class ManualSynchronizationRequiredException(override val message: String) : RuntimeException(message) {
    companion object {
        @Serial
        private const val serialVersionUID: Long = -8664623010021064227L
    }
}